neo/rawio/tdtrawio.py

```diff
@@ -439,6 +439,8 @@ def _rescale_event_timestamp(self, event_timestamps, dtype):
     ('CircType', 'int'),
     ('NumChan', 'int'),
     ('StrobeMode', 'int'),
+    ('StrobeBuddy', 'S4'),
+    ('SecTag','S4'),
```

Comment:
I am afraid that this two new lines are totally version dependent.
Could you investigate this ?